sentry-opentelemetry/sentry-opentelemetry-agentcustomization/src/main/java/io/sentry/opentelemetry/SentryAutoConfigurationCustomizerProvider.java

```diff
@@ -4,19 +4,87 @@
 import io.opentelemetry.sdk.autoconfigure.spi.AutoConfigurationCustomizerProvider;
 import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
 import io.opentelemetry.sdk.trace.SdkTracerProviderBuilder;
+import io.sentry.Instrumenter;
+import io.sentry.Sentry;
+import io.sentry.SentryOptions;
+import io.sentry.protocol.SdkVersion;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Enumeration;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.jar.Attributes;
+import java.util.jar.Manifest;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 public final class SentryAutoConfigurationCustomizerProvider
     implements AutoConfigurationCustomizerProvider {
 
   @Override
   public void customize(AutoConfigurationCustomizer autoConfiguration) {
+    final @Nullable String sentryPropertiesFile = System.getenv("SENTRY_PROPERTIES_FILE");
+    final @Nullable String sentryDsn = System.getenv("SENTRY_DSN");
+
+    if (sentryPropertiesFile != null || sentryDsn != null) {
+      Sentry.init(
```

Comment:
so the benefit of that is that people can just include our agent and use it without any additional config in their app, right?

What if they are already using Sentry and just want to include the agent, but then we'd double-init, which isn't great?

Reply:
Yeah, also thought about that. My idea was to add another ENV var, that allows users to turn off the auto init. Can do that as a new PR once all the open ones are merged. It's already on the TODO list here: https://github.com/getsentry/sentry-java/issues/2327

Reply:
yeah that'd make sense. Similar to AndroidManifest 👍